File: Contadordecafe/app/src/main/java/com/esampaio/apps/contadordecafe/view/estatistica/activity/ChartActivity.java
package com.esampaio.apps.contadordecafe.view.estatistica.activity;

import android.graphics.Color;
import android.support.design.widget.TabLayout;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.FrameLayout;
import android.widget.Toast;

import com.esampaio.apps.contadordecafe.R;
import com.esampaio.apps.contadordecafe.view.estatistica.fragment.FragmentGraficos;
import com.github.mikephil.charting.charts.HorizontalBarChart;
import com.github.mikephil.charting.charts.LineChart;
import com.github.mikephil.charting.charts.PieChart;
import com.github.mikephil.charting.components.Description;
import com.github.mikephil.charting.data.BarData;
import com.github.mikephil.charting.data.BarDataSet;
import com.github.mikephil.charting.data.BarEntry;
import com.github.mikephil.charting.data.Entry;
import com.github.mikephil.charting.data.LineData;
import com.github.mikephil.charting.data.LineDataSet;
import com.github.mikephil.charting.data.PieData;
import com.github.mikephil.charting.data.PieDataSet;
import com.github.mikephil.charting.data.PieEntry;

import java.util.ArrayList;
import java.util.List;

public class ChartActivity extends AppCompatActivity {

    private TabLayout tabLayout;
    private FrameLayout content;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.estatisticas_activity);
        tabLayout = (TabLayout) findViewById(R.id.tablayout);
        tabLayout.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
            @Override
            public void onTabSelected(TabLayout.Tab tab) {
                setContent(tab.getPosition());
            }

            @Override
            public void onTabUnselected(TabLayout.Tab tab) {}

            @Override
            public void onTabReselected(TabLayout.Tab tab) {}
        });
        content = (FrameLayout) findViewById(R.id.fragment_container);
    }
    private void setContent(int tabSelected){
        FragmentGraficos fragmentGraficos = FragmentGraficos.newInstance();
        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, fragmentGraficos).commit();
    }
}

File: Contadordecafe/app/src/main/java/com/esampaio/apps/contadordecafe/view/estatistica/activity/ChartActivity.kt
package com.esampaio.apps.contadordecafe.view.estatistica.activity

import android.os.Bundle
import android.support.design.widget.TabLayout
import android.support.v7.app.AppCompatActivity
import com.esampaio.apps.contadordecafe.R
import com.esampaio.apps.contadordecafe.view.estatistica.fragment.FragmentGraficos
import com.esampaio.apps.contadordecafe.view.estatistica.fragment.GraficosDiarioFragment

class ChartActivity : AppCompatActivity() {
    private var tabLayout: TabLayout? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.estatisticas_activity)
        tabLayout = findViewById(R.id.tablayout) as TabLayout
        tabLayout!!.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
            override fun onTabSelected(tab: TabLayout.Tab) {
                setContent(tab.position)
            }
            override fun onTabUnselected(tab: TabLayout.Tab) {}

            override fun onTabReselected(tab: TabLayout.Tab) {}
        })
        setContent(0);
    }

    private fun setContent(tabSelected: Int) {
        var fragmentGraficos:FragmentGraficos;
        if(tabSelected==0){
            fragmentGraficos = GraficosDiarioFragment.newInstance()
        }else {
           fragmentGraficos = FragmentGraficos.newInstance()
        }
        supportFragmentManager.beginTransaction().replace(R.id.fragment_container, fragmentGraficos).commit()
    }
}
